src/oncall/constants.py

```diff
@@ -42,7 +43,9 @@ def init(config):
     global DEFAULT_MODES
     global DEFAULT_TIMES
     global SUPPORTED_TIMEZONES
+    global GRACE_PERIOD
     DEFAULT_ROLES = config['notifications']['default_roles']
     DEFAULT_MODES = config['notifications']['default_modes']
     DEFAULT_TIMES = config['notifications']['default_times']
-    SUPPORTED_TIMEZONES = config['supported_timezones']
\ No newline at end of file
+    SUPPORTED_TIMEZONES = config['supported_timezones']
+    GRACE_PERIOD = config.get('grace_period', 86400)
```

Comment:
Do we want to default it to a day, or default it to `0` to restore old behavior?